Assets/_Application/DataTypeAdapters/SubObjectColorCSVImportAdapter.cs

```diff
@@ -34,7 +34,7 @@ public void Execute(LocalFile localFile)
             
             activeCartesianTileSubObjectColorLayer = Instantiate(layerGameObjectPrefab);
```

Comment:
Dit gaat nog via de oude methode van layers instantieren waardoor je hieronder de LayerData uit de property ophaalt - zou dit niet ook gewoon via de Layers service moeten lopen? Dan heb je deze workaround niet nodig hieronder

Reply:
klopt, CSV is in main ook stuk, dus die hebben we even genegeerd totdat die in de fix ticket wordt opgepakt (1869). Ik zal hier een todo bij zetten